doc/Installation/Installation-Ubuntu-1804-Apache.md

```diff
@@ -0,0 +1,154 @@
+source: Installation/Installation-Ubuntu-1804-Apache.md
+> NOTE: These instructions assume you are the **root** user.  If you are not, prepend `sudo` to the shell commands (the ones that aren't at `mysql>` prompts) or temporarily become a user with root privileges with `sudo -s` or `sudo -i`.
+
+**Please note the minimum supported PHP version is 5.6.4**
+
+## Install Required Packages ##
+
+    apt update
+    apt install apache2 composer fping git graphviz imagemagick libapache2-mod-php7.2 mariadb-client mariadb-server mtr-tiny nmap php7.2-cli php7.2-curl php7.2-gd php7.2-json php7.2-mysql php7.2-snmp php7.2-xml php7.2-zip python-memcache python-mysqldb rrdtool snmp snmpd whois
```

Comment:
missing mbstring, please update this so it is based of the latest install docs for 16.04.